logicrunner/casebind.go

```diff
@@ -73,31 +73,48 @@ func NewCaseBindFromExecutorResultsMessage(msg *message.ExecutorResults) *CaseBi
 	panic("not implemented")
 }
 
-func (cb *CaseBind) ToValidateMessage(ctx context.Context, ref Ref, pulse core.Pulse) *message.ValidateCaseBind {
-	res := &message.ValidateCaseBind{
-		RecordRef: ref,
-		Requests:  make([]message.CaseBindRequest, len(cb.Requests)),
-		Pulse:     pulse,
+func (cb *CaseBind) getCaseBindRequestsFromMessageBus(ctx context.Context) []message.CaseBindRequest {
+	requests := make([]message.CaseBindRequest, 0)
```

Comment:
This doesn't look right. The slice length used to be `len(cb.Requests)` and below you access it as `requests[i]`.

Reply:
fixed